### PR TITLE
Fixed Span N-Plus-1

### PR DESCRIPTION
**Pull request by asafchen-dig**

### CODE REVIEW

File: src/main/kotlin/org/digma/intellij/plugin/ui/list/insights/SpanNPlusOneInsightPanel.kt
Comment (arik-dig) [inline src/main/kotlin/org/digma/intellij/plugin/ui/list/insights/SpanNPlusOneInsightPanel.kt:1]: please reformat the code

File: model/src/main/kotlin/org/digma/intellij/plugin/model/rest/insights/EndpointInfo.kt [hunk 1]
```diff
@@ -7,11 +7,12 @@ import java.beans.ConstructorProperties
...
-@ConstructorProperties("route", "serviceName", "instrumentationLibrary", "codeObjectId","spanCodeObjectId")
+@ConstructorProperties("route", "serviceName", "instrumentationLibrary", "codeObjectId","spanCodeObjectId", "entrySpanCodeObjectId")
 constructor(
     val route: String,
     val serviceName: String,
     val instrumentationLibrary: String,
     val codeObjectId: String?,
-    val spanCodeObjectId:String
+    val spanCodeObjectId:String,
+    val entrySpanCodeObjectId:String?, // Nullable for BC
```
Comment (arik-dig) [inline model/src/main/kotlin/org/digma/intellij/plugin/model/rest/insights/EndpointInfo.kt:17]: can you link in this PR to the backend PR so we could see them linked?

**Review by arik-dig — APPROVED** [review]
i guess it relates to issue #831?
please connect them

File: src/main/kotlin/org/digma/intellij/plugin/ui/list/insights/SpanNPlusOneInsightPanel.kt
Comment (asafchen-dig) [reply src/main/kotlin/org/digma/intellij/plugin/ui/list/insights/SpanNPlusOneInsightPanel.kt:1]: done

**Comment by asafchen-dig** [comment]
Related to digma-ai/digma-collector-backend#977

File: model/src/main/kotlin/org/digma/intellij/plugin/model/rest/insights/EndpointInfo.kt (hunk 1 above)
Comment (asafchen-dig) [reply model/src/main/kotlin/org/digma/intellij/plugin/model/rest/insights/EndpointInfo.kt:17]: done

**Pull request merged**